defaults.env

```diff
@@ -3,3 +3,4 @@ PULSE_DOMAIN=www.mozillapulse.org
 NETWORK_API_DOMAIN=network.mofoprod.net
 VIRTUAL_ROOT=/
 TARGET_DOMAIN=network.mofostaging.net
+CSP=default-src 'self'; script-src 'self' cdn.optimizely.com; style-src 'self' code.cdn.mozilla.net fonts.googleapis.com; img-src * data:; font-src 'self' fonts.gstatic.com fonts.googleapis.com code.cdn.mozilla.net; connect-src 'self' *.mozilla.org; media-src 'self'; object-src 'self'; child-src 'self'; frame-src 'self';
```

Comment:
if we add `'sha256-vcOkZrGWRfmgv+69QFqtI4SpPCnNHq6JIzLcQvzg5/8='` to `script-src` we will not have to disable live reload for development. that hash is tha SHA256 fingerprint of the live reload script.